swift/ql/lib/codeql/swift/dataflow/ExternalFlow.qll

```diff
@@ -390,16 +390,12 @@ predicate matchesSignature(AbstractFunctionDecl func, string signature) {
   paramsString(func) = signature
 }
 
-private NominalType getDeclType(IterableDeclContext decl) {
-  result = decl.(ClassDecl).getType()
+private NominalTypeDecl resolveExtensions(IterableDeclContext decl) {
+  // TODO: this should be a method on IterableDeclContext
```

Comment:
If we're happy with the approach I will move this predicate into `IterableDeclContext`.

Reply:
Done.